### PR TITLE
Grants backend complete

### PR DESCRIPTION
**Pull request by varortz**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

### :cloud: :cloud: Description :cloud: :cloud:
[//]: # "REQUIRED - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- Fixed the formatting for the grants' `deadline` field
- Added a new screen (GrantInfo) to replace the modal pop-up after clicking on a grant for additional information. This involved editing `userStack.tsx` and `navigation.tsx`
- Removed hard-coded data for grants.

### Screenshots :calling:
[//]: # "REQUIRED for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! If you are making changes here, please CC: @leexesther at the bottom."

## :zap: How to review :zap:
[//]: # 'REQUIRED - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'
These changes can be reviewed all at once. Since no styling has been added yet, simply check whether the grant info is formatted correctly both on the grants feed and the new GrantInfo page.  

## :seedling: Next steps :seedling:
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

- Support uploading images for grants (admin)
- Display images for grants on the Grants page and GrantInfo page
- Update the GrantsCard component to reflect the new design changes
- Add icons for grant fields displayed on the grant cards
- Add the react native component for sharing grant applications  (`share-link`)
- Implement a "new grant" notifier feature  

### Online sources :information_source:
[//]: # 'Optional - copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs :raised_hands:
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

[//]: # 'This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
CC: @davidqing6432

### CODE REVIEW

File: src/components/GrantCard.tsx [hunk 1]
```diff
@@ -22,7 +13,7 @@ const styles = StyleSheet.create({
   cardContainer: {
     flexDirection: 'column',
     alignItems: 'flex-start',
-    backgroundColor: '#C94420',
+    backgroundColor: '#CC502F',
```
Comment (david-qing) [inline src/components/GrantCard.tsx]: use colors from our global colors

File: src/navigation/userStack.tsx [hunk 2]
```diff
@@ -1,11 +1,13 @@
 import { NavigationContainer } from '@react-navigation/native';
 import { createStackNavigator } from '@react-navigation/stack';
+import QueriesDemo from '../firebase/QueriesDemo';
```
Comment (david-qing) [inline src/navigation/userStack.tsx]: remove queriesDemo from userStack.

File: src/types/schema.ts [hunk 3]
```diff
@@ -19,8 +19,10 @@ export type Grant = {
   grant_id: string;
   amount: number;
   countries: string[];
-  deadline: Date;
+  deadline: string;
```
Comment (david-qing) [inline src/types/schema.ts:22]: is changing it to deadline correct here? does this work properly?

File: src/types/navigation.tsx [hunk 4]
```diff
@@ -11,6 +12,7 @@ export type RootStackParamList = {
   Grants: undefined;
   Play: undefined;
   QueriesDemo: undefined;
+  GrantInfo: { grantObj: Grant };
```
Comment (david-qing) [inline src/types/navigation.tsx]: we might want to add this granting page into the grantsstackparamlist, instead of rootstack

File: src/screens/GrantInfoScreen/styles.tsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#FFF',
```
Comment (david-qing) [inline src/screens/GrantInfoScreen/styles.tsx]: use global colors

File: src/screens/GrantInfoScreen/styles.tsx [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+  container2: {
+    flex: 1,
+    backgroundColor: '#FFF',
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: '100%',
+  },
+  header_text: {
+    fontSize: 13,
+    width: '90%',
```
Comment (david-qing) [inline src/screens/GrantInfoScreen/styles.tsx]: try to use pixels instead of percentages

Comment (david-qing) [reply src/screens/GrantInfoScreen/styles.tsx]: for all percentages, unless they are 100%

**Review by david-qing — APPROVED** [review]
looks good to merge!
